internal/objects/validator_test.go

```diff
@@ -721,6 +721,12 @@ func (fakeNSClient) RESTMapper() meta.RESTMapper {
 func (fakeNSClient) Scheme() *runtime.Scheme {
 	return nil
 }
+func (fakeNSClient) GroupVersionKindFor(_ runtime.Object) (schema.GroupVersionKind, error) {
+	return schema.GroupVersionKind{}, nil
+}
+func (fakeNSClient) IsObjectNamespaced(_ runtime.Object) (bool, error) {
+	return true, nil
```

Comment:
No client in `fakeNSClient`, so these still return dummy values. Not sure if `true` is a good default for this function.

Reply:
Seems like an ok default to me